src/main/res/layout/card_tooltip.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView
```

Comment:
Is this file used or is it leftover from a previous attempt w/ a different library?